CMakeLists.txt

```diff
@@ -61,11 +64,15 @@ target_link_libraries( ${LRS_TARGET} PUBLIC rsutils )
 if(BUILD_WITH_DDS)
     if (CMAKE_SYSTEM MATCHES "Windows" OR CMAKE_SYSTEM MATCHES "Linux")
 
+        pop_security_flags()
```

Comment:
Can we place the push/pop inside the projects instead?
This way it will be places at the same context as the project and if anoter cmake will include it it will get it automatically